### DISCUSSION TITLE
How do I install a preset and get obsidian behaving like shown in the README.md file.

### DISCUSSION THREAD

**Discussion by TheGreatApollyon** [Akifyss/obsidian-border#209] [Q&A] [answered]
Just as the question suggests, installing a preset is pretty confusing for *me*. I'd appreciate if you could guide me through it

I did import it through style-settings plugin, but it doesn't even look remotely similar to the one shown in the preview neither am I getting the auto-hide side-panel  animations. It just looks like default 

I'm using .deb version on Ubuntu 23.10

**Comment by Akifyss** [comment, marked as answer]
The preset file only affects the ”Appearance“ section, auto-hide is a preference setting that needs to be manually enabled.

**Reply by TheGreatApollyon** [reply]
Even the appearance was almost unchanged when I imported the preset. For reference I imported the translucent preset, but it was looking very similar to the default theme without any transparency in the app

**Reply by Akifyss** [reply]
Oh, I see. Your system is Linux. Both Linux and Windows systems do not support enabling transparent windows.
![image](https://github.com/Akifyss/obsidian-border/assets/23567154/ece703d6-8796-41f0-a322-ee44346e2e37)

**Reply by TheGreatApollyon** [reply]
Oh, that's a bummer. Thanks anyways

**Reply by TheGreatApollyon** [reply]
Also, I'd appreciate if you could tell where I can find the auto-hide sidebar setting/plugin. I was able to find neither

**Reply by Akifyss** [reply]
![image](https://github.com/Akifyss/obsidian-border/assets/23567154/e073ec7b-657e-4b2c-8891-3b5ac5388433)
